Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -966,4 +967,32 @@ infoCardView config orientation' infoCardConfig =
         , visibility infoCardConfig.subHeading2.visibility
         ] <> infoCardConfig.subHeading2.fontStyle
         ]
-    ]
\ No newline at end of file
+    ]
+
+parkingChargesView :: forall w. Config -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+parkingChargesView config push = 
+  linearLayout
+  [ width MATCH_PARENT
+  , height WRAP_CONTENT
+  , margin $ MarginBottom 8
+  , gravity CENTER
+  , visibility $ boolToVisibility config.topCard.showParkingCharge
+  ]
+  [ textView $
+    [ text "P"
```

Comment:
What's this P for?

Reply:
Added it according to this design's [P]:
<img width="821" alt="image" src="https://github.com/nammayatri/nammayatri/assets/83444620/c8273351-6407-4959-a5c6-4672e612c66b">